hub/api/dataset.py

```diff
@@ -140,6 +140,45 @@ def empty(
             storage=cache_chain, read_only=read_only, public=public, token=token
         )
 
+    @staticmethod
+    def exists(
+        path: str,
+        memory_cache_size: int = DEFAULT_MEMORY_CACHE_SIZE,
```

Comment:
we don't need the cache size arguments for this function, you can just pass these constants to get_storage_and_cache_chain